orchagent/flexcounterorch.cpp

```diff
@@ -247,3 +248,46 @@ bool FlexCounterOrch::bake()
     Consumer* consumer = dynamic_cast<Consumer *>(getExecutor(CFG_FLEX_COUNTER_TABLE_NAME));
     return consumer->addToSync(entries);
 }
+
+bool FlexCounterOrch::isQueueIndexValid(int index)
+{
+    return (MIN_QUEUE_INDEX <= index && index <= MAX_QUEUE_INDEX);
+}
+
+bool FlexCounterOrch::getQueueConfigurations(std::string portName, bool *queuesState)
+{
+    vector<FieldValueTuple> portFields;
+    m_counterQueueConfigTable.get(portName, portFields);
+    for (auto& portField : portFields)
+    {
+        auto field = fvField(portField);
+        auto value = fvValue(portField);
+        if (field == "queue_list")
+        {
+            auto configQueuesState = tokenize(value, ',');
+            for (auto& queue : configQueuesState)
+            {
+                std::stringstream queueStream(queue);
+                int queueIndex;
+                queueStream >> queueIndex;
```

Comment:
stringstream does not let you know if the string is successfully parsed. We have a utility function in swss common:
```suggestion
                int queueIndex{};
                try {
                     queueIndex = swss::to_int<int>(queue);
                } catch (std::invalid_argument) {
                      continue;
                }
```

Reply:
Done